examples/otlptrace/src/main/java/com/google/cloud/opentelemetry/example/otlptrace/OTLPTraceExample.java

```diff
@@ -63,6 +64,18 @@ private static SpanExporter addAuthorizationHeaders(
       } catch (IOException e) {
         System.out.println("error:" + e.getMessage());
       }
+    } else if (exporter instanceof OtlpGrpcSpanExporter) {
+      try {
+        credentials.refreshIfExpired();
+        OtlpGrpcSpanExporterBuilder builder =
```

Comment:
Do we have the ability to directly pass a credentials option?  https://github.com/grpc/grpc-java/blob/34e241a60e7dd54195daaec3a16decae0dd702cc/api/src/main/java/io/grpc/CallOptions.java#L137

It might be nice to open an issue upstream if that isn't possible today

Reply:
I think gRPC's concept of `CallOptions` is not exposed to the end user in OTel Java's OTLP exporters.

That being said, I think closest analog to `WithCallCredentials` is the concept of an [`Authenticator`](https://github.com/open-telemetry/opentelemetry-java/blob/9e12dd60c457ddbe0f5a178fc9790f7a22c1e926/exporters/common/src/main/java/io/opentelemetry/exporter/internal/auth/Authenticator.java#L19) which can be applied to both OTLPGrpcExporter and OTLPHttpExporter. 

However this class is in the internal (experimental) package currently.

Reply:
Cool.  It might be worth opening an issue upstream to see if they can accept CallCredentials in some form.